.github/workflows/Ray_CI.yml

```diff
@@ -0,0 +1,93 @@
+name: Ray.jl - CI
```

Comment:
Can we rename the other CI workflow to include the package name just so these match?

Reply:
eh, given that we're gonna split this out I think this is better (to make that separation less painful)